pom.xml

```diff
@@ -151,11 +152,7 @@
                         <artifactId>logback-classic</artifactId>
                     </exclusion>
                 </exclusions>
-            </dependency>
-            <dependency>
-                <groupId>org.apache.kafka</groupId>
-                <artifactId>kafka-server-common</artifactId>
-                <version>${kafka.version}</version>
+                <scope>provided</scope>
```

Comment:
Should we mark kafka_2.13 as an `optional` dependency, in addition to `provided`? 

I confess I've never fully got to grips with the concept of optional dependen cies and when one should and should not use them